"oidc" missing when running operator locally

When using `make run-local` command from local against the latest NightShift2 cluster, there will an error occur `panic: No Auth Provider found for name "oidc"`.
The whole stack trace lists as follows:
```
operator-sdk up local --operator-flags -webhook=false
INFO[0000] Running the operator locally.                
INFO[0000] pravega-operator Version:                    
INFO[0000] Git SHA:                                     
INFO[0000] Go Version: go1.10.4                         
INFO[0000] Go OS/Arch: linux/amd64                      
INFO[0000] operator-sdk Version: v0.4.0                 
panic: No Auth Provider found for name "oidc"

goroutine 1 [running]:
github.com/pravega/pravega-operator/vendor/k8s.io/client-go/discovery.NewDiscoveryClientForConfigOrDie(0xc4204d8a80, 0xc4204c9a60)
	/home/wenqi/Desktop/go/src/github.com/pravega/pravega-operator/vendor/k8s.io/client-go/discovery/discovery_client.go:454 +0x65
github.com/pravega/pravega-operator/vendor/sigs.k8s.io/controller-runtime/pkg/client/apiutil.NewDiscoveryRESTMapper(0xc4204d8a80, 0x1000000012ab93b, 0x17, 0x8, 0x4349ab)
	/home/wenqi/Desktop/go/src/github.com/pravega/pravega-operator/vendor/sigs.k8s.io/controller-runtime/pkg/client/apiutil/apimachinery.go:35 +0x2f
github.com/pravega/pravega-operator/vendor/sigs.k8s.io/controller-runtime/pkg/manager.New(0xc4204d8a80, 0xc4201c6e00, 0x1323eb8, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42003c450, ...)
	/home/wenqi/Desktop/go/src/github.com/pravega/pravega-operator/vendor/sigs.k8s.io/controller-runtime/pkg/manager/manager.go:156 +0x167
main.main()
	/home/wenqi/Desktop/go/src/github.com/pravega/pravega-operator/cmd/manager/main.go:90 +0x2bb
exit status 2
FATA[0002] failed to run operator locally: (exit status 1) 
Makefile:53: recipe for target 'run-local' failed
make: *** [run-local] Error 1
```

The problem lies in the `main.go` file, it needs to import `	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"` to enable `oidc` auth.

--- a/cmd/manager/main.go
+++ b/cmd/manager/main.go
@@ -26,6 +26,7 @@ import (
 	controllerconfig "github.com/pravega/pravega-operator/pkg/controller/config"
 	"github.com/pravega/pravega-operator/pkg/version"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"

